modules/comment/comment.permissions.js

```diff
@@ -47,8 +47,8 @@ const commentPermissionsMutations = {
     inputDataValidation(REPLY_COMMENT_DATA, replyCommentUpdateValidator),
     hasRoles([ADMIN, SUPERADMIN])
   ),
-  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
-  deleteReplyForComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
+  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN, USER])),
```

Comment:
For me, it looks like any member with role USER can remove any comment.

Reply:
What is the purpose of rule hasRoles([ADMIN, SUPERADMIN, USER]) here if it always returns true? Mb you don't need to change this line?

Reply:
So, what about `or( and(isTheSameUser, hasRoles[USER]),  hasRoles([ADMIN, SUPERADMIN]))` ?

Reply:
It was fixed on the frontend part and no need for changes on the backend. 
You could close this PR.